src/stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,214 @@
+//SDS ALERT
+.sds-alert {
+  padding: 5px;
+  border: solid 0.1rem #fbc036;
+  border-radius: 0.5rem;
+  width: 175px;
+
+  .sds-alert-icon {
+    display: inline-block;
+    text-align: center;
+    height: 25px;
+    width: 25px;
+    color: #ffffff;
+    background-color: #fbc036;
+    border-radius: 50%;
+    padding-top: 1px;
+    margin: auto 5px;
+  }
+}
+
+//SDS ALERT INFO
+.sds-alert.info {
+  padding: 5px;
+  border: solid 0.1rem #00BDE3;
+  border-radius: 0.5rem;
+  width: 175px;
```

Comment:
The common styles are repeated for each type, maybe something like this could make it more DRY?

```

.sds-alert {
  // common styles

  .sds-alert-icon {
    // common styles
  }

  &.info {
    border-color: 'infobg' 
    
    .sds-alert-icon {
      background-color: 'infocolor';
    }
  }

  &.error {
    border-color: 'errorbg' 
    
    .sds-alert-icon {
      background-color: 'errorcolor';
    }
  }
}
```